ldk/javascript/src/keyboard/keyboard.test.ts

```diff
@@ -20,16 +20,16 @@ describe('Keyboard', () => {
             };
 
             keyboard.listenHotkey(hotkey, callback);
-            expect(oliveHelps.keyboard.listenHotkey).toHaveBeenCalledWith(hotkey, callback);
+            expect(oliveHelps.keyboard.listenHotkey).toHaveBeenCalledWith(hotkey, callback,expect.any(Function));
         });
 
-        it('throws exception when olive helps call fails', () => {
+        it('rejects with the error when the underlying call throws an error', () => {
```

Comment:
Wondering if this is still valid after the refactor. All what listenHotkey() doing is calling promisifyListenableWithParam(). Perhaps such test belong in the promisify.test.ts. In this case it will only be tested once and not from every aptitude which depends on this call. (More of the unit test approach.)

Reply:
I think it depends on how deeply we want these tests to operate. These tests, after the refactor, are no longer technically unit tests b/c they are also testing the promisify functions. However, keeping them as-is would still validate that the whole chain is interacting with the OH functions as expected.